index.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Produce</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<!--- THIS IS CARD 1 -->
+	<!-- <div class="flextop"> -->
```

Comment:
Since you're not using this div, I'd remove it and un-indent all the following tags